docs/account_access.md

```diff
@@ -31,3 +33,15 @@ The account security level. Different levels have access to different commands.
 ### RealmID
 
 The [Realm ID](realmlist#id).
+
+### comment
+
+```sql
+CREATE TABLE IF NOT EXISTS `account_access` (
+  `id` int(10) unsigned NOT NULL,
+  `gmlevel` tinyint(3) unsigned NOT NULL,
+  `RealmID` int(11) NOT NULL DEFAULT '-1',
+  `comment` varchar(255) DEFAULT '',
+  PRIMARY KEY (`id`,`RealmID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+```
```

Comment:
Why?

Reply:
> Why?

Add the SQL so that it just corroborates with the information in the table.

Reply:
But it does not add any use, this is just the same info that you give in the table at the top of the page

Reply:
> But it does not add any use, this is just the same info that you give in the table at the top of the page

Perfect, we remove it then.